### PR TITLE
Fix styles in the demos

### PR DESCRIPTION
**Pull request by blasten**
cc @notwaldorf

### CODE REVIEW

File: demo/hide-header.html [hunk 1]
```diff
@@ -22,9 +22,13 @@
   <link rel="import" href="../../paper-toolbar/paper-toolbar.html">
   <link rel="import" href="../../iron-icons/iron-icons.html">
   <link rel="import" href="../../paper-icon-button/paper-icon-button.html">
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
+
+  <link rel="import" href="../../paper-styles/color.html">
+  <link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
   <link rel="import" href="sample-content.html">
 
+  <link rel="stylesheet" href="../../paper-styles/demo.css">
```
Comment (notwaldorf) [inline demo/hide-header.html]: While you're at it, you should move this to Polymer 1.1.0, and use a style import

File: demo/index.html [hunk 2]
```diff
@@ -68,6 +70,10 @@
       padding: 8px;
     }
 
+    .flex {
```
Comment (notwaldorf) [inline demo/index.html]: super minor nit: in the toolbar demo I actually called this `spacer`, which kind of makes more sense, imho, and makes it look less like you're just copy pasting the flex class.

**Comment by notwaldorf** [comment]
LGTM with a tiny nit you can ignore.

**Comment by blasten** [comment]
@notwaldorf `spacer` sounds good

**Pull request merged**